workspaces/global-header/plugins/global-header/src/components/MenuItemLink/MenuItemLink.tsx

```diff
@@ -38,6 +38,8 @@ export const MenuItemLink = ({
   icon,
   tooltip,
 }: MenuItemLinkProps) => {
+  const isExternalLink = to.startsWith('http') || to.startsWith('https');
```

Comment:
```suggestion
  const isExternalLink = to.startsWith('http');
```

`https` starts with `http`